content/html/concepts/semantic-html/terms/mark/mark.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '<mark>'
+Description: "Represents part of a text that should be rendered as marked or highlighted, due to its being particularly relevant in its specific context. It can be used to draw the reader's attention to a specific word, phrase or section."
```

Comment:
```suggestion
Description: 'Represents part of a text that should be rendered as marked or highlighted, due to its being particularly relevant in its specific context. It can be used to draw the reader's attention to a specific word, phrase or section.'
```